mudserver.py

```diff
@@ -149,6 +149,7 @@ def _check_for_new_connections(self):
             self._new_events.append(
                 Event(ServerEvents.NEW_PLAYER, client)
             )
+            print(self._new_events)
```

Comment:
We should probably get logging going soon.

Reply:
Aye didn't mean to leave that in really